src/Cli/ConfigGenerator.cs

```diff
@@ -681,6 +686,99 @@ private static bool TryUpdateDepthLimit(
+                }
+                else
+                {
+                    return false;
+                }
+            }
+
+            return runtimeConfig != null;
+        }
+
+        /// <summary>
+        /// Attempts to update the Config parameters in the GraphQL runtime settings based on the provided value.
+        /// Validates that any user-provided parameter value is valid and then returns true if the updated GraphQL options
+        /// needs to be overwritten on the existing config parameters
+        /// </summary>
+        /// <param name="options">options.</param>
+        /// <param name="updatedGraphQLOptions">updatedGraphQLOptions.</param>
+        /// <returns>True if the value needs to be udpated in the runtime config, else false</returns>
+        private static bool UpdateConfigureGraphQLValues(
+            ConfigureOptions options,
+            ref GraphQLRuntimeOptions? updatedGraphQLOptions)
+        {
+            object? updatedValue;
+
+            // Runtime.GraphQL.Enabled
+            updatedValue = options?.RuntimeGraphQLEnabled;
+            if (updatedValue != null)
+            {
+                updatedGraphQLOptions = updatedGraphQLOptions! with { Enabled = (bool)updatedValue };
+                _logger.LogInformation($"Updated RuntimeConfig with Runtime.GraphQL.Enabled as '{updatedValue}'");
+            }
+
+            // Runtime.GraphQL.Path
+            updatedValue = options?.RuntimeGraphQLPath;
+            if (updatedValue != null)
+            {
+                bool status = RuntimeConfigValidatorUtil.TryValidateUriComponent((string)updatedValue, out string exceptionMessage);
+                if (status)
+                {
+                    updatedGraphQLOptions = updatedGraphQLOptions! with { Path = (string)updatedValue };
+                    _logger.LogInformation($"Updated RuntimeConfig with Runtime.GraphQL.Path as '{updatedValue}'");
+                }
+                else
+                {
+                    _logger.LogError($"Failure in updating RuntimeConfig with Runtime.GraphQL.Path " +
+                        $"as '{updatedValue}' due to exception message: {exceptionMessage}");
+                    return false;
+                }
+            }
+
+            // Runtime.GraphQL.Allow-Introspection
+            updatedValue = options?.RuntimeGraphQLAllowIntrospection;
+            if (updatedValue != null)
+            {
+                updatedGraphQLOptions = updatedGraphQLOptions! with { AllowIntrospection = (bool)updatedValue };
+                _logger.LogInformation($"Updated RuntimeConfig with Runtime.GraphQL.AllowIntrospection as '{updatedValue}'");
+            }
+
+            // Runtime.GraphQL.Multiple-mutations.Create.Enabled
+            updatedValue = options?.RuntimeGraphQLMultipleMutationsCreateEnabled;
+            if (updatedValue != null)
+            {
+                MultipleCreateOptions multipleCreateOptions = new((bool)updatedValue);
```

Comment:
Is this a convention followed throughout? We are calling constructors and functions, so should i add parameter key in each occurrence? Because i feel this is only needed if we want to pass only some parameters and not all of them else its easily inferable.

Reply:
In this case, either the named parameter or change `updatedValue` var to something like `updatedEnabledValue` to distinguish what the var represents.

Reply:
That variable is common for all so cant change the name. Modified the enabled: for this scenario